maximum_subarray/C/max_subarray.c

```diff
@@ -0,0 +1,29 @@
+#include<stdio.h>
+
+int n,arr[200005],ans,curSum;
+
+int main(){
+	// Reading size of array and array.
+	scanf("%d",&n);
+	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp458ja7w6/maximum_subarray/C/max_subarray.c
+++ b/tmp/tmp458ja7w6/maximum_subarray/C/max_subarray.c
@@ -5,7 +5,7 @@
 int main(){
 	// Reading size of array and array.
 	scanf("%d",&n);
-	
+
 	for(int i=1;i<=n;i++)
 		scanf("%d",&arr[i]);
 
```